src/stravalib/model.py

```diff
@@ -747,7 +668,7 @@ class name, photo type, and a key identifier."""
         else:
             photo_type = "(no type)"
             idfield = "id"
-            idval = self.id
+            idval = self.uid
```

Comment:
Mypy (rightfully) complained about `id` not existing in this class. Somehow, it passed in previous versions for unknown reasons.